### PR TITLE
Feature Read

### PR DESCRIPTION
**Pull request by Kaylen7**
Added VC of `/view` route.
Resolved conflicts with styles.css. 

@AlexP953 try to test it out in your pc to see if routes work correctly. You can also try filtering and search features 🔥

### CODE REVIEW

File: app/views/scripts/components/html/sideMenu.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<!-- MENU -->
```
Comment (AlexP953) [inline app/views/scripts/components/html/sideMenu.html]: Maybe we could use this component for all views that have the sidebar, making its content optional between this and the back button?

Comment (Kaylen7) [reply app/views/scripts/components/html/sideMenu.html]: Good thinking! I'll refactor it tomorrow 👍

Comment (Kaylen7) [reply app/views/scripts/components/html/sideMenu.html]: @AlexP953  I have created the component. Take a look at the files. 

Uses are: 
**Go back menu**
```php
new sideMenuComponent('back');       
```
**Main menu**
```php
new sideMenuComponent('main'); //empty strings or other strings also render this menu.
```

**Comment by Kaylen7** [comment]
@AlexP953 wait before merging, I want to try to improve accessibility to `editView` menu on tasks.

**Comment by Kaylen7** [comment]
@AlexP953 I think I've added all features required for this branch. After your revision, feel free to merge this branch 🔥

**Pull request merged**